library/src/main/java/com/getstream/sdk/chat/adapter/viewholder/attachment/AttachmentViewHolder.kt

```diff
@@ -142,23 +142,9 @@ class AttachmentViewHolder(
         }
         mediaBinding.tvMediaDes.text = firstAttachment.text
 
-        if (firstAttachment.text.isNullOrEmpty()) {
-            mediaBinding.tvMediaDes.isVisible = false
-        } else {
-            mediaBinding.tvMediaDes.isVisible = true
-        }
-
-        if (firstAttachment.title.isNullOrEmpty()) {
-            mediaBinding.tvMediaTitle.isVisible = false
-        } else {
-            mediaBinding.tvMediaTitle.isVisible = true
-        }
-
-        if (type == ModelType.attach_video) {
-            mediaBinding.tvMediaPlay.isVisible = true
-        } else {
-            mediaBinding.tvMediaPlay.isVisible = false
-        }
+        mediaBinding.tvMediaDes.isVisible = !firstAttachment.text.isNullOrEmpty()
+        mediaBinding.tvMediaTitle.isVisible = !firstAttachment.title.isNullOrEmpty()
+        mediaBinding.tvMediaPlay.isVisible = type == ModelType.attach_video
```

Comment:
Since it seems like `type` is nullable, maybe let's do the java trick to avoid NPE?
```suggestion
        mediaBinding.tvMediaPlay.isVisible = ModelType.attach_video == type
```

Reply:
Kotlin's `==` operator is null-safe by default ([see docs](https://kotlinlang.org/docs/reference/equality.html)):

>Structural equality is checked by the == operation (and its negated counterpart !=). By convention, an expression like a == b is translated to:

```kotlin
a?.equals(b) ?: (b === null)
```